apps/frontend/src/pages/root/components/user-wheel/libs/constants/constants.ts

```diff
@@ -0,0 +1,3 @@
+const WHEEL_DATE_FORMAT = "d MMM yyyy, EEEE";
```

Comment:
It will be typed in `getFormattedDate`, no need to have a separate constant for that